pvlib/test/test_solarposition.py

```diff
@@ -367,3 +367,15 @@ def test_nrel_earthsun_distance():
     expected = pd.Series(np.array([0.983289204601]),
                          index=pd.DatetimeIndex([times, ]))
     assert_series_equal(expected, result)
+
```

Comment:
shouldn't we add tests for all of the other functions you've added to ``solarposition.py``? I think they should be marked private if you don't want to add the tests.